### DISCUSSION TITLE
2024-06-17(월)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#155] [데일리스크럼]
아래 내용을 복사해 작성해주세요.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by Seoin02** [comment]
### 완료한 일
- 온보딩 API 연결
- 회원가입 로직 수정
- 회원 탈퇴 구현

### 해야할 일
- my 페이지 새로고침 이슈 해결
- 주문 상세 페이지 컴포넌트 분리
- 프로필 페이지 API 연결
- 닉네임 중복 검사 연결(#146)

### 어려운 일
- 멘토님 피드백을 반영했으나 my 페이지 새로고침 이슈를 아직 해결하지 못했습니다...

**Comment by devwqc** [comment]
### 완료한 일
- BottomSheet, Modal 리팩토링 완료
- CardSlider 컴포넌트 구현(오전 회의 종료 후 PR 올릴 예정)

### 해야할 일
- 페이지 작업 시작(홈, 검색, 찜)

### 어려운 일
- 파이팅!! 💪☺️

**Comment by seolsis** [comment]
### 완료한 일
- 장바구니 api
### 해야할 일
- 결제 api
- lottie 파일로 변경
- 배송메시지
- 상품추천컴포넌트연결
### 어려운 일

**Comment by q45402sk** [comment]
### 완료한 일
deliveryEditPage

### 해야할 일
myDeliveryPage
purchaseDeliveryPage
상품상세 스크롤 적용

### 어려운 일
deliveryEditPage 우편번호 에러처리

**Comment by minyoung0503** [comment]
### 완료한 일

### 해야할 일
my -> review page API 연결
상품 옵션 BottomSheet 

### 어려운 일
